sdk/src/loom2d/Display/Stage.ls

```diff
@@ -430,6 +431,11 @@ package loom2d.display
                     }
 
                     break;
+                case StageScaleMode.MANUAL:
+                    scaleX = scaledWidth;
+                    scaleY = scaledHeight;
+                    x = 0;
+                    y = 0;
```

Comment:
@bengarney can you look at this? I think @MarkFassett wanted to modify `x` and `y` manually as well and this might override it? I think the scale is ok, but I'm not 100% on the expected usage.